core/src/main/scala/caliban/introspection/adt/__Type.scala

```diff
@@ -102,3 +102,10 @@ case class __Type(
       case _                   => true
     }
 }
+
+object __Type {
+  implicit class __TypeExtensions(val it: __Type) {
+    def list: __Type    = __Type(__TypeKind.LIST, ofType = Some(it))
+    def nonNull: __Type = __Type(__TypeKind.NON_NULL, ofType = Some(it))
+  }
+}
```

Comment:
Just curious, why do we need to make these into type extensions instead of just putting them on the `__Type` class? Is it a bincompat thing?

Reply:
Oh yeah, didn't realize that 😆

Reply:
🤦 Aye, good point